main.js

```diff
@@ -0,0 +1,31 @@
+function Add(numbers = "") {
+    if (numbers.trim() == "") return 0;
+    let self = numbers.trim().split(',')
+    if (self.length > 2) {
+        throw new Error("Exceeded Numbers")
+    }
+    let sum = 0;
+
+    let negStack = []
+    
+    for (item of self) {
+        if (isNaN(item) === true) {
+            throw new Error("Wrong Numbers")
+        } else {
+            if (Number(item) < 0 )
+            {
+                negStack.push(Number(item))
+            }
+            if (Number(item)<=1000)
+            sum += Number(item);
+        }
+    }
+    if (negStack.length > 0){
```

Comment:
can you please describe what is the complexity of your code? 

and can you think of a better way of writing it?